src/store/models/Community.js

```diff
@@ -44,5 +45,7 @@ export const DEFAULT_BANNER = 'https://d3ngex8q79bk55.cloudfront.net/misc/defaul
 export const DEFAULT_AVATAR = 'https://d3ngex8q79bk55.cloudfront.net/misc/default_community_avatar.png'
 
 export const ALL_COMMUNITIES_ID = 'all-communities'
-
 export const ALL_COMMUNITIES_AVATAR_PATH = '/assets/white-merkaba.png'
+
+export const PUBLIC_COMMUNITIES_ID = 'public-communities'
```

Comment:
i would call this PUBLIC_POSTS_ID='public' or something since its not about showing posts from public communities, its about showing all public posts. but not a big deal

Reply:
changed to PUBLIC_CONTEXT_ID='public-context'